lib/bolt/config.rb

```diff
@@ -101,6 +102,14 @@ class Config
       "level"  => "`warn` for console, `notice` for file"
     }.freeze
 
+    APPLY_SETTINGS = {
+      "show_diff" => "Whether to log and report a contextual diff when files are being replaced. See https://puppet.com/docs/puppet/latest/configuration.html#showdiff"
```

Comment:
```suggestion
      "show_diff" => "Whether to log and report a contextual diff when files are being replaced. See the [documenation](https://puppet.com/docs/puppet/latest/configuration.html#showdiff)."
```

Not using the URL will look a bit nicer when this is rendered as markdown.